protocol/indexer/shared/order_removal_reason.go

```diff
@@ -30,6 +30,8 @@ func ConvertOrderRemovalReasonToIndexerOrderRemovalReason(
 		reason = sharedtypes.OrderRemovalReason_ORDER_REMOVAL_REASON_IMMEDIATE_OR_CANCEL_WOULD_REST_ON_BOOK
 	case clobtypes.OrderRemoval_REMOVAL_REASON_VIOLATES_ISOLATED_SUBACCOUNT_CONSTRAINTS:
 		reason = sharedtypes.OrderRemovalReason_ORDER_REMOVAL_REASON_VIOLATES_ISOLATED_SUBACCOUNT_CONSTRAINTS
+	case clobtypes.OrderRemoval_REMOVAL_REASON_PERMISSIONED_KEY_EXPIRED:
+		reason = sharedtypes.OrderRemovalReason_ORDER_REMOVAL_REASON_EXPIRED
```

Comment:
note: I am just re-using `OrderRemovalReason_ORDER_REMOVAL_REASON_EXPIRED` here so that no indexer changes are required and the word `EXPIRED` is also kind of appropriate here. 

@teddyding let me know if you feel strongly that we should define a separate reason for this.

Reply:
Makes sense as a shortcut, could we add inline comment to provide more context?